index.bs

```diff
@@ -144,7 +144,8 @@ The only currently valid `Cross-Origin-Embedder-Policy` value is "`require-corp`
 
 In order to support forward-compatibility with as-yet-unknown request types, user agents MUST ignore
 this header if it contains an invalid value. Likewise, user agents MUST ignore this header if the
-value cannot be parsed as a <a grammar>`sh-token`</a>.
+value cannot be parsed as a <a grammar>`sh-token`</a>. Parameters are not allowed; user agents MUST
+ignore this header if any parameters are present.
```

Comment:
Are these requirements novel or do they duplicate the processing model? I kinda hope the latter in which case this should be a note.

Reply:
They duplicate the processing model. Marking them as a note is fine; they aren't the normative description.

Reply:
I've changed MUSTs to will. That makes this a statement of fact. Do you think it should be a note?

Reply:
Seems fine either way, though I do worry a tiny bit about readers that aren't careful.

Reply:
Changed to a note.